src/content/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data.mdx

```diff
@@ -177,6 +177,10 @@ You can [query your infrastructure data](/docs/using-new-relic/data/understand-d
 SELECT * FROM ProcessSample
 ```
 
+Metrics from the New Reic Infrastructure Agent (SystemSample, StorageSample, NetworkSample and ProcessSample) are aggregated into buckets, in order to improve the query performance over longer periods of time.
+NRQL queries for under 1 hour of time will scan the raw data, and longer queries will scan 1 minute / 10 minute / 1 hour and 3 hour aggregated buckets.
+Until October/November 2024 these aggregated event buckets only contained 1 average value per metric. Starting in October/November 2024 the aggregation has been improved to use the [gauge format](/docs/data-apis/understand-data/metric-data/metric-data-type/) for accurate differentiation between `max(), min() and average()`. 
```

Comment:
@nr-ksteinbach Can you specify dates here? If not, I'd prefer to say "Until November 2024..."

Reply:
answered in slack.